lib/containers/utils.pm

```diff
@@ -179,4 +179,20 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+=head2 can_build_sle_base
+
+C<can_build_sle_base> should be used to identify if sle base image runs against a
+system that it does not support registration and SUSEConnect.
+In this case the build of the container engine is not going to work as the base images
+lacks of the repos.
+
+The call should return false if you try to test sle base container on osd but host is not sle
+true otherwise.
+
+=cut
+sub can_build_sle_base {
+    my $has_sle_registration = script_run("SUSEConnect --status-text | grep Containers");
```

Comment:
This is good idea! cc: @grisu48 @ge0r 
```bash
SLES12-SP3 # # SUSEConnect --status-text | grep Containers; echo $?
  Containers module
0
SLES15-SP2 # # SUSEConnect --status-text | grep Containers; echo $?
  Containers module
0
```

Reply:
I thought about that as well, but `SUSEConnect --status-text` can be incredibly slow as it makes multiple API calls.

Reply:
This is true, but in addition it also checks that the host has the Containers module activated.

Reply:
That doesn't actually matter - as long as the system is registered with SLE*, that's enough. `container-suseconnect` handles the activation of the modules itself. So just `test -e /etc/zypp/credentials.d/SCCcredentials` would work as well.

Reply:
That is very true!

Reply:
done